### DISCUSSION TITLE
asreview not found

### DISCUSSION THREAD

**Discussion by testori** [asreview/asreview#911] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
Dear all, 
I have installed asreview on my mac (M1), see below the message:

Requirement already satisfied: asreview in ./Library/Python/3.7/lib/python/site-packages (0.19)
Requirement already satisfied: setuptools in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from asreview) (40.8.0)
Requirement already satisfied: numpy in ./Library/Python/3.7/lib/python/site-packages (from asreview) (1.17.2)
Requirement already satisfied: pandas in ./Library/Python/3.7/lib/python/site-packages (from asreview) (0.25.1)
Requirement already satisfied: h5py in ./Library/Python/3.7/lib/python/site-packages (from asreview) (3.6.0)
Requirement already satisfied: flask-cors in ./Library/Python/3.7/lib/python/site-packages (from asreview) (3.0.10)
Requirement already satisfied: flask>=2.0 in ./Library/Python/3.7/lib/python/site-packages (from asreview) (2.0.2)
Requirement already satisfied: dill in ./Library/Python/3.7/lib/python/site-packages (from asreview) (0.3.4)
Requirement already satisfied: rispy~=0.7.0 in ./Library/Python/3.7/lib/python/site-packages (from asreview) (0.7.1)
Requirement already satisfied: gevent in ./Library/Python/3.7/lib/python/site-packages (from asreview) (21.12.0)
Requirement already satisfied: openpyxl in ./Library/Python/3.7/lib/python/site-packages (from asreview) (3.0.9)
Requirement already satisfied: sklearn in ./Library/Python/3.7/lib/python/site-packages (from asreview) (0.0)
Requirement already satisfied: xlrd>=1.0.0 in ./Library/Python/3.7/lib/python/site-packages (from asreview) (2.0.1)
Requirement already satisfied: click>=7.1.2 in ./Library/Python/3.7/lib/python/site-packages (from flask>=2.0->asreview) (7.1.2)
Requirement already satisfied: itsdangerous>=2.0 in ./Library/Python/3.7/lib/python/site-packages (from flask>=2.0->asreview) (2.0.1)
Requirement already satisfied: Werkzeug>=2.0 in ./Library/Python/3.7/lib/python/site-packages (from flask>=2.0->asreview) (2.0.2)
Requirement already satisfied: Jinja2>=3.0 in ./Library/Python/3.7/lib/python/site-packages (from flask>=2.0->asreview) (3.0.3)
Requirement already satisfied: Six in ./Library/Python/3.7/lib/python/site-packages (from flask-cors->asreview) (1.12.0)
Requirement already satisfied: greenlet<2.0,>=1.1.0 in ./Library/Python/3.7/lib/python/site-packages (from gevent->asreview) (1.1.2)
Requirement already satisfied: zope.event in ./Library/Python/3.7/lib/python/site-packages (from gevent->asreview) (4.5.0)
Requirement already satisfied: zope.interface in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from gevent->asreview) (4.6.0)
Requirement already satisfied: cached-property in ./Library/Python/3.7/lib/python/site-packages (from h5py->asreview) (1.5.2)
Requirement already satisfied: et-xmlfile in ./Library/Python/3.7/lib/python/site-packages (from openpyxl->asreview) (1.1.0)
Requirement already satisfied: pytz>=2017.2 in ./Library/Python/3.7/lib/python/site-packages (from pandas->asreview) (2019.2)
Requirement already satisfied: python-dateutil>=2.6.1 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pandas->asreview) (2.6.1)
Requirement already satisfied: scikit-learn in ./Library/Python/3.7/lib/python/site-packages (from sklearn->asreview) (1.0.2)
Requirement already satisfied: MarkupSafe>=2.0 in ./Library/Python/3.7/lib/python/site-packages (from Jinja2>=3.0->flask>=2.0->asreview) (2.0.1)
Requirement already satisfied: scipy>=1.1.0 in ./Library/Python/3.7/lib/python/site-packages (from scikit-learn->sklearn->asreview) (1.3.1)
Requirement already satisfied: joblib>=0.11 in ./Library/Python/3.7/lib/python/site-packages (from scikit-learn->sklearn->asreview) (1.1.0)
Requirement already satisfied: threadpoolctl>=2.0.0 in ./Library/Python/3.7/lib/python/site-packages (from scikit-learn->sklearn->asreview) (3.0.0)

However, when I try to launch asreview with asreview lab, I receive the following error:
zsh: command not found: asreview

Can someone help me with that?

Thanks a lot!

**Comment by Rensvandeschoot** [comment]
Have you tried the solution in this [thread](https://github.com/asreview/asreview/issues/738)?

**Reply by testori** [reply]
Hi,
Yes, I have tried the solution in the thread but I cannot install miniforge as several errors pop-up.
Also it seems that asreview is installed but then when I call it it says that "command not found", so I am really confused.

**Comment by ottomattas** [comment, marked as answer]
Dear @testori,

First of all, thank you for opening this discussion and providing also some details.

Could you maybe report this as an issue here: https://github.com/asreview/asreview/issues/new/choose
This will help you formalise the situation a bit better and then give us more to work with hopefully.

That said, there could be many reasons for this happening. I have some ideas which we could put to test after you have opened an issue as suggested above.
